pmm/main.go

```diff
@@ -143,11 +143,11 @@ Use 'pmm-admin config' to define server user and password.`, a.Config.ServerAddr
 	}
 
 	// Check Consul status.
-	if leader, err := a.consulAPI.Status().Leader(); err != nil || leader != "127.0.0.1:8300" {
```

Comment:
Why we have this condition like `leader != "127.0.0.1:8300"` why not `leader == ""`? I wasn't able to mock consul with [httptest](https://golang.org/pkg/net/http/httptest/) because of this condition. This also would fix #22 - user wants to use external consul.